packages/synapse-react-client/src/components/EntityFinder/details/view/DetailsView.tsx

```diff
@@ -78,36 +109,271 @@ export type EntityFinderTableViewRowData = EntityFinderHeader & {
   currentSelectedVersion?: number
 }
 
+const columnHelper = createColumnHelper<EntityFinderTableViewRowData>()
+
+function NameRenderer(props: {
+  context: CellContext<EntityFinderTableViewRowData, string>
+  setCurrentContainer?: (containerId: string) => void
+}) {
+  const { context, setCurrentContainer } = props
+  const { row, getValue } = context
+  const rowName = getValue()
+  if (setCurrentContainer && isContainerType(row.original.entityType)) {
+    return (
+      <span
+        role="link"
+        className="EntityFinderTableCellContainerLink"
+        onClick={e => {
+          e.stopPropagation()
+          setCurrentContainer(row.original.id)
+        }}
+      >
+        <Tooltip title={rowName}>
+          <span className="nameColumnCell">{rowName}</span>
```

Comment:
Should these cell renderers be using MUI Typography?

Reply:
(and if so, how much overhead does that introduce in the DOM?)

Reply:
(was worried about scalability in the DOM, but remembered this is virtualized)

Reply:
They could, but I don't think it affects the end result. I tend to use Typography when I need alternate variants or the content would benefit from standard margins/line height, neither of which are necessary here